tests/skyserve/llm/service.yaml

```diff
@@ -1,7 +1,10 @@
 service:
-  readiness_probe: /v1/models
+  readiness_probe: /${API_VERSION}/models
```

Comment:
It will be great to use sth similar to the original issue (#3048) in our smoke test, as it is more widely used and could test env vars in service/run section in the same time. Considering use `MODEL_NAME` for `lmsys/fastchat-t5-3b-v1.0`

Reply:
Good suggestion, just updated!